frequency-chain/style-guide
# 17 form dropdown

## Goal

The goal of this PR is to create a dropdown item and storybook.

Closes #17 #21 

## Discussion

- install shadcn dropdown component
  - also did some package updates
- create dropdown component
- style to match Figma
- create storybook with different cases

## Checklist

- [n/a] Docs updated
- [n/a] Tests added

## How To Test the Changes

1. Clone the pr branch
2. run storybook
3. see select component working as expected

## Demo

https://github.com/user-attachments/assets/8af17ef8-0e6f-43dd-bf9a-d77809a9fc75
